core/misc/form.js

```diff
@@ -5,7 +5,7 @@
  */
 $.fn.backdropGetSummary = function () {
   var callback = this.data('summaryCallback');
-  return (this[0] && callback) ? $.trim(callback(this[0])) : '';
+  return (this[0] && callback) ? callback(this[0]).trim() : '';
```

Comment:
I could see certain situations where this may throw an error. If a summary callback incorrectly returns `false` or does not return anything at all (`null`), then `(false).trim()` or `(null).trim()` will throw a console error. `$.trim(null)` returns an empty string.

There's probably a shorter/cleaner way to write this, but this would be more robust against incorrect return values:
```suggestion
  var returnValue = (this[0] && callback) ? callback(this[0]) : '';
  return typeof returnValue === 'string' ? returnValue.trim() : '';
```